Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -458,13 +458,13 @@ export default class TimelineView extends Vue {
                     </b-row>
                 </div>
                 <LinearTimeline
-                    v-show="isLinearView && !isLoading"
+                    v-if="isLinearView && !isLoading"
```

Comment:
This might cause some problems when rendering since both components sync when changing pages.

Reply:
This looks like is breaking the calendar view on the first load or when the linear view changes [1]. Checked and it does not look like the timeline and calendar view are in sync anymore
[1] ![image](https://user-images.githubusercontent.com/12201292/125980717-e6805e40-567c-45eb-9fe4-49096cc8fa1e.png)

Reply:
Will fix as discussed.